frontend/lib/features/onboarding/onboarding_welcome_page.dart

```diff
@@ -43,6 +46,22 @@ class OnboardingWelcomePage extends StatelessWidget {
                 child: Text(Loc.of(context).getStarted),
               ),
             ),
+            if (kDebugMode) const SizedBox(height: Grid.md), // Add some spacing
+            if (kDebugMode) Padding(
+              padding: const EdgeInsets.symmetric(horizontal: Grid.side),
```

Comment:
Do you want to keep these `kDebugMode` mode cases in here?

Reply:
yeah not sure how/where else to do it yet (just needed a way to flag things so you can click on it, and debug seemed easy)

Reply:
Cool. I think we should remove this from the `WelcomePage` for now and we can add the real page where this will be used soon.